### ISSUE TITLE
[App]: custom plots no longer displayed while corresponding table is OK

### ISSUE THREAD

**Issue by svandenhaute** [wandb/wandb#5334]
### Current Behavior

When calling `wandb.plot.scatter` based on an existing `Table` instance, the corresponding figure is generated in the web app but is rendered empty with a message saying 'no data available' (see screenshot). The corresponding `Table` instance is correctly logged, with all values that are required for the plot being present in the table.
![image](https://user-images.githubusercontent.com/45051541/231744771-6eac53ce-4d6b-4c96-bdef-70a537feb333.png)

### Expected Behavior

The custom scatter plot with dots corresponding to the rows in the table.

### Steps To Reproduce

```
table = wandb.Table(columns=columns, data=data)
plot  = wandb.plot.scatter(table, columns[0], columns[1], title='foo')
wandb.log({'table': table, 'plot': plot})
```

### Environment

OS: `linux-popos-skylake`

Browsers: Chrome

Version: `0.14.2`

**Comment by danielnbarbosa** [comment]
Also experiencing this.  The plots eventually render after about 30 mins.

**Comment by nate-wandb** [comment]
Hi All, could someone send a link to a run where they see this?

I believe it is related to a current issue we have processing files that's listed on our status page [here](https://status.wandb.com/). The runs likely don't have the necessary files uploaded to pull the data from.

**Comment by svandenhaute** [comment]
https://wandb.ai/svandenhaute/psiflow/runs/fqrywi4o/overview?workspace=user-svandenhaute though I'm not sure this is publicly accessible?

**Comment by guyfreund** [comment]
I'm facing this issue as well, sadly can't share the run as it's private

**Comment by nate-wandb** [comment]
Thank you @svandenhaute. Yes, I can confirm this is caused by the file processing issue. Unfortunately, this is still ongoing as we are working with the GCS team to get a resolution on this. I can update here once this is resolved but file uploads are currently being delayed several hours. We'll also keep the [status page](https://status.wandb.com/) up to date if you'd like to subscribe to alerts

**Comment by nate-wandb** [comment]
Hi all, this should be resolved now. I checked @svandenhaute's workspace and charts are loading now. Sorry for the inconvenience and let us know if you still see any issues.

Thank you,
Nate

**Comment by paksas** [comment]
I'm experiencing this issue too right now.

The link to my run: https://wandb.ai/ptrochim/distilgpt2-for-game?workspace=user-ptrochim

For this project, I was logging only plots at first, and the  I wanted to add a table with text columns. As soon as I did, the plots stopped showing, and only the table remained.
When I removed table logging, reverting back to the former code, the new runs started showing an empty table only - still no plots.
